digitalassetlinks/src/test/java/com/solana/digitalassetlinks/AndroidAppPackageVerifierUnitTests.java

```diff
@@ -205,11 +223,11 @@ public void testAppPackageVerificationNoMatchingPackageInPackageManager() {
     private static PackageManager mockPackageManagerFactory(@NonNull String packageName,
                                                             @NonNull byte[][] certificates,
                                                             boolean multipleSigners) {
-        if (certificates.length == 0) {
-            throw new IllegalArgumentException("at least 1 certificate required");
-        } else if (multipleSigners && certificates.length == 1) {
-            throw new IllegalArgumentException("multipleSigners requires at least 2 certificates");
-        }
+        // if (certificates.length == 0) {
+        //     throw new IllegalArgumentException("at least 1 certificate required");
+        // } else if (multipleSigners && certificates.length == 1) {
+        //     throw new IllegalArgumentException("multipleSigners requires at least 2 certificates");
+        // }
```

Comment:
```suggestion
        // NOTE: empty certificates would normally be an error, but we want to exercise unit tests
        // for this case, so allow it when constructing a mock PackageManager
```

Adding a comment here to explain why the mock package manager allows this (rather than leaving the code in place but commented out, which might be confusing to future readers.